packages/eslint-config-vtex-react/gatsby.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  extends: ['./index.js'],
+  overrides: [
+    {
+      files: ['gatsby-{ssr,browser,node,config}.{js,jsx,ts,tsx}'],
+      rules: {
+        'global-require': 'off',
+        '@typescript-eslint/no-require-imports': 'off',
+        '@typescript-eslint/no-var-requires': 'off',
+      },
+    },
+    {
+      files: ['gatsby-{node,config,ssr}.{js,jsx,ts,tsx}'],
```

Comment:
I believe that browser should be on the list too
```suggestion
      files: ['gatsby-{ssr,browser,node,config}.{js,jsx,ts,tsx}'],
```